### DISCUSSION TITLE
support .net6.0

### DISCUSSION THREAD

**Discussion by gmf520** [canton7/Stylet#325] [General]
I hope it will come soon

**Comment by canton7** [comment]
Stylet already supports .NET 6 - you can use it just fine in an application which targets .NET 6.

I don't think there's any point in multi-targeting every new .NET release -- that will just bloat the nuget packet with loads of completely unnecessary DLLs.

So what problem specifically are you trying to solve?

(The templates default to generating .NET 5 which could do with an update)

**Comment by gmf520** [comment]
Yes, It is a problem with my code and the support is not necessary, tks!
